src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
Actually in umd mode where globals are ready and automatic registeration happend. Users do call this function directly so we still need to save the module somewhere. Best location is the component itself scince its already global.